extensions/empty-screenshots/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { readdir, stat, rm } from "fs/promises";
+import { join, resolve } from "path";
+import { homedir } from "os";
+import { showToast, Toast, showHUD, getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  daysToKeep: string;
+  screenshotFolder: string;
+}
+
+export default async function main() {
+  const { daysToKeep, screenshotFolder } = getPreferenceValues<Preferences>();
+  const numberOfDays = parseInt(daysToKeep, 10);
+  const folder = resolve(screenshotFolder.replace("~", homedir));
+
+  const files = await readdir(folder);
+
+  const screenshots = files
+    .filter((file) => file.endsWith(".png") || file.endsWith(".mov"))
```

Comment:
Could it be an idea to make it a preference which filetypes should be removed? You could have a comma-separated list of extensions (with .mov/.png as default selected)

Reply:
Sure!